sambamba/slice.d

```diff
@@ -283,17 +287,24 @@ int slice_main(string[] args) {
 
     // at least two arguments must be presented
     string output_filename = null;
+    string bed_filename = null;
 
     try {
         getopt(args,
                std.getopt.config.caseSensitive,
-               "output-filename|o", &output_filename);
+               "output-filename|o", &output_filename,
+               "regions|L",         &bed_filename);
 
-        if (args.length < 3) {
+        if ( (bed_filename.length == 0 && args.length < 3) ||
```

Comment:
It would be more readable to first check if `args.length < 2`, then put `args[2..$]` into a variable and then check if it's empty/non-empty.